spark-3/src/test/scala/org/neo4j/spark/ReauthenticationIT.scala

```diff
@@ -38,7 +37,12 @@ object ReauthenticationIT {
 
   private val KEYCLOAK = new KeycloakContainer("quay.io/keycloak/keycloak:26.2.5")
     .withNetwork(NETWORK).withNetworkAliases("keycloak")
-    .useTlsKeystore("/neo4j-keycloak.jks", "testpwd")
+    .withCopyFileToContainer(
```

Comment:
question: I assume this is because `useTlsKeystore` is not available in `1.9.0`, right?